packages/broker/src/plugins/storage/DataQueryFormat.ts

```diff
@@ -1,5 +1,5 @@
 import { StreamMessage } from '@streamr/protocol'
-import { convertBytesToStreamMessage } from '@streamr/trackerless-network'
+import { convertBytesToStreamMessage } from '@streamr/sdk'
```

Comment:
Can we get rid of `@streamr/trackerless-network` from broker's package.json?

Reply:
Not yet, as `StreamID` functionality is there